src/timeMachine/actions/queries.ts

```diff
@@ -370,6 +385,52 @@ export const executeQueries =
     }
   }
 
+export const runDownloadQuery = () => async (dispatch, getState: GetState) => {
+  const state = getState()
+  const queryText = getActiveQuery(state).text
+
+  try {
+    await dispatch(hydrateVariables())
```

Comment:
variable hydration occurs in 3 places (as of right now):
* when executing a timeMachine query
* when downloading a csv from timeMachine.
   * (note: this can be done without running a query first)
* when opening the variables toolbar.

Since the csv download can occur seperately (without a precursor) of the other 2 steps, then IMO we do need to hydrate here.

Reply:
Makes sense! Thank you for explaining that to me :)